src/Process/3dTilesProcessing.js

```diff
@@ -165,13 +234,17 @@ export function process3dTilesNode(cullingTest, subdivisionTest) {
             if ((node.material === undefined || node.material.visible)) {
```

Comment:
Unrelated to current PR, but why do we set `visible = false` for all children (on line 236) when `node.material.visible == true` ? I would have expected the contrary.

Reply:
This code looks really fishy...
@Jeremy-Gaillard can you explain what is this code doing?

Reply:
This is strange indeed. `node.material` is always undefined, so the condition is useless.

Logically, we should be hiding only the nodes that we will not be visiting. So, the code from line 235 to 240 should be at line 231. I tested it on your branch and it works, I'll push a fix.